pygdf/series.py

```diff
@@ -409,6 +412,40 @@ def dtype(self):
         """dtype of the Series"""
         return self._impl.dtype
 
+    @classmethod
+    def _concat(cls, objs, index=True):
+        head = objs[0]
+        for o in objs:
+            if o._impl != head._impl:
+                raise ValueError("All series must be of same type")
+
+        data = head._impl.concat(objs)
```

Comment:
I don't think we need to delegate to `_impl` to handle `concat`.  The GPU dataframe will always require plain-old-data types.  (We can only handle data that can be copied b/w cpu and gpu memory.)  It is always safe to just copy the data for concat just like the one in numerical.